app/components/Poll/index.tsx

```diff
@@ -51,197 +51,243 @@ type OptionEntityRatio = OptionEntity & {
+              <Tooltip content="Trykk for mer info">{title}</Tooltip>
+            ) : (
+              <>{title}</>
             )}
           </Flex>
-        )}
-        {!poll?.hasAnswered && (
-          <Flex column className={styles.optionWrapper}>
-            {!expanded && (
-              <Flex
-                alignItems="center"
-                className={styles.blurContainer}
-                onClick={toggleTruncate}
-              >
-                <p className={styles.blurOverlay}>
-                  Klikk her for å se alle alternativene
-                </p>
-              </Flex>
-            )}
-            {options &&
-              optionsToShow?.map((option) => (
-                <Flex
-                  justifyContent="space-between"
-                  style={{ flexGrow: '1' }}
-                  className={cx(expanded ? '' : styles.blurEffect)}
-                  key={option.id}
-                >
-                  <Button
-                    dark
-                    onClick={() => dispatch(votePoll(poll.id, option.id))}
-                    className={styles.voteButton}
-                  >
-                    {option.name}
-                  </Button>
-                </Flex>
-              ))}
-          </Flex>
-        )}
-        <div>
-          <div className={styles.moreOptionsLink}>
-            {truncateOptions &&
-              (!poll?.hasAnswered ||
-                !poll?.resultsHidden ||
-                allowedToViewHiddenResults) && (
-                <Flex alignItems="center" justifyContent="center">
-                  <Icon
-                    onClick={toggleTruncate}
-                    name={expanded ? 'chevron-up' : 'chevron-down'}
-                    size={20}
-                  />
-                </Flex>
-              )}
-          </div>
-          <Flex justifyContent="space-between">
+        </Link>
+      </Flex>
+      <Flex
+        column
+        className={styles.contentWrapper}
+        style={{
+          height: alwaysOpen ? `auto` : expanded ? `${expandedHeight}px` : '0',
+        }}
+      >
+        <div ref={optionRef}>
+          {canAnswer && (
+            <VoteOpen details={details} poll={poll} options={options} />
+          )}
+          {!canAnswer && showResults && (
+            <VoteResults
+              details={details}
+              poll={poll}
+              options={options}
+              resultsHidden={resultsHidden}
+            />
+          )}
+          {!canAnswer && !showResults && (
+            <VoteHidden details={details} poll={poll} />
+          )}
+
+          <Flex
+            alignItems="center"
+            justifyContent="center"
+            className={styles.registrationCount}
+          >
+            <Tooltip content="Avstemningen er anonym.">
+              <Icon
+                name="information-circle-outline"
+                size={17}
+                style={{ cursor: 'pointer', margin: '0 5px' }}
```

Comment:
Is this margin still needed?

Reply:
Yup - but it should've been in the css file